docs/impl_defined/schemas/config_pkg.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2024 Thales DIS France SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Original Author: Zbigniew CHAMSKI - Thales
+
+# Schema of a config package file
+
+from dataclasses import dataclass
+from typing import List, Union, Optional
+
+# Enumerate config struct fields
```

Comment:
These fields are defined in config_pkg.sv.
They should be read from that file to avoid redundancy, to avoid mismatches.